set.mm

```diff
@@ -477620,6 +477788,376 @@ The sumset operation can be used for both group (additive) operations and
       WEGUIWGIXPHVTUMWAWB $.
   $}
 
+  ${
+    $d .(+) x $.  $d .0. a o x $.  $d A a o x $.  $d B a o x $.  $d G a o x $.
+    grplsm0l.b $e |- B = ( Base ` G ) $.
+    grplsm0l.p $e |- .(+) = ( LSSum ` G ) $.
+    grplsm0l.0 $e |- .0. = ( 0g ` G ) $.
+    $( Sumset with the identity singleton is the original set.  (Contributed by
+       Thierry Arnoux, 27-Jul-2024.) $)
+    grplsm0l $p |- ( ( G e. Grp /\ A C_ B /\ A =/= (/) )
+                  -> ( { .0. } .(+) A ) = A ) $=
+      ( vx vo va cgrp wcel wss c0 co cv wceq wrex eqeq2d wne w3a csn cplusg cfv
+      wb grpidcl snssd lsmelvalx 3expa an32s mpidan 3adant3 simpl1 simp2 sselda
+      eqid grplid syl2anc equcom syl6bb rexbidva c0g fvexi oveq1 rexbidv risset
+      wa rexsn 3bitr4g bitrd eqrdv ) DLMZABNZAOUAZUBZIEUCZACPZAVPIQZVRMZVSJQZKQ
+      ZDUDUEZPZRZKASZJVQSZVSAMZVMVNVTWGUFZVOVMVNVQBNZWIVMEBBDEFHUGUHVMWJVNWIVMW
+      JVNWIJKBWCCVQADLVSFWCUQZGUIUJUKULUMVPVSEWBWCPZRZKASZWBVSRZKASWGWHVPWMWOKA
+      VPWBAMZVHZWMVSWBRWOWQWLWBVSWQVMWBBMWLWBRVMVNVOWPUNVPABWBVMVNVOUOUPBWCDWBE
+      FWKHURUSTIKUTVAVBWFWNJEEDVCHVDWAERZWEWMKAWRWDWLVSWAEWBWCVETVFVIKVSAVGVJVK
+      VL $.
+
+    $( Sumset with the identity singleton is the original set.  (Contributed by
+       Thierry Arnoux, 27-Jul-2024.) $)
+    subglsm0l $p |- ( ( G e. Grp /\ A e. ( SubGrp ` G ) )
```

Comment:
Maybe "Sumset of a subgroup with the identity singleton is the original subgroup."

I think the hypothesis  `` B = ( Base ` G ) `` is not needed here, and the antecedent `G e. Grp`is also not required (follows from ``A e. ( SubGrp ` G )`` by ~issubg).

Reply:
...and then this theorem becomes exactly [`~lsm02`](https://us.metamath.org/mpeuni/lsm02.html)!
Thanks, many functions operating on structures work on `_V`, and I never noticed `SubGrp` only accepts groups.
I'll also apply that "trick" to `~nsgqus0` just above, and now I notice hypotheses in others of my previous theorems can be simplified, too. I'll do that as I encounter them.

Reply:
Done! I've completely removed this theorem in favor of `~lsm02`, and reworked the other theorems in this PR to remove the `G e. Grp` hypothesis.